api/src/org/labkey/api/query/CrosstabView.java

```diff
@@ -110,7 +108,7 @@ protected DataRegion createDataRegion()
     @Override
     public List<DisplayColumn> getDisplayColumns()
     {
-        assert getTable() instanceof CrosstabTableInfo;
+        assert getTable() instanceof CrosstabTableInfo && ((CrosstabTableInfo) getTable()).isCrosstab();

```

Comment:
I think you could do:

```suggestion
        assert getTable() instanceof CrosstabTableInfo cti && cti.isCrosstab();

```